docker-compose.yml

```diff
@@ -37,5 +38,29 @@ services:
       # DOZZLE_USERNAME=
       # DOZZLE_PASSWORD=
 
+  ocrd-database:
+    image: "mongo:latest"
+    container_name: ocrd-database
+
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${DB_ROOT_USER:-root}
+      MONGO_INITDB_ROOT_PASSWORD: ${DB_ROOT_PASSWORD:-root_password}
+
+    volumes:
+      - db-volume:/data/db
+
+
+  mongo-express:
+    image: mongo-express:latest
+    depends_on: 
+      - ocrd-database
+    ports:
+      - 8081:8081
```

Comment:
Native port 8081 is occupied on my server. Perhaps we should make this configurable as `MONITOR_PORT_DBE`?

Reply:
see https://github.com/slub/ocrd_monitor/commit/f5db85d65d88221d4667e7baade8ce0a3f181c1c